inc-setup/export.php

```diff
@@ -57,3 +59,63 @@ function _mw_adminimize_export_json() {
 	echo wp_json_encode( $settings );
 	exit();
 }
+
+function _mw_adminimize_export_role_json() {
```

Comment:
Please add a phpdoc part. short description, parameter and return type, content.